setup.py

```diff
@@ -46,6 +46,6 @@ def load_requirements(*requirements_paths):
         'Programming Language :: Python',
     ],
     packages=['submissions', 'submissions.migrations'],
-    install_requires=load_requirements('requirements.txt'),
+    install_requires=load_requirements('requirements.txt').append('Django<1.9a0'),
```

Comment:
I'm working to make sure this syntax is recognized by gemnasium, will update when I get more details.